core/src/mindustry/mod/Mods.java

```diff
@@ -611,7 +611,7 @@ private LoadedMod loadMod(Fi sourceFile) throws Exception{
 
         Fi metaf = zip.child("mod.json").exists() ? zip.child("mod.json") : zip.child("mod.hjson").exists() ? zip.child("mod.hjson") : zip.child("plugin.json");
         if(!metaf.exists()){
-            Log.warn("Mod {0} doesn't have a 'mod.json'/'plugin.json'/'mod.js' file, skipping.", sourceFile);
+            Log.warn("Mod {0} doesn't have a 'mod.json'/'plugin.json'/'mod.hjson' file, skipping.", sourceFile);
```

Comment:
```suggestion
            Log.warn("Mod {0} doesn't have a 'mod.json'/'mod.hjson'/'plugin.json' file, skipping.", sourceFile);
```
this lists them in the order they are checked for ^